webui/react/config-overrides.js

```diff
@@ -52,5 +54,26 @@ module.exports = override(
       'process.env.VERSION': '"0.12.12.dev0"',
       'process.env.IS_DEV': JSON.stringify(process.env.NODE_ENV === 'development'),
     })
-  )
+  ),
+
+  addWebpackPlugin(
+    new MonacoWebpackPlugin({
+      // available options are documented at https://github.com/Microsoft/monaco-editor-webpack-plugin#options
+      languages: ['yaml'],
+      features: [
+        'suggest',
+        'quickOutline',
+        'wordHighlighter',
+        'colorDetector',
+        'codelens',
+        'parameterHints'
+      ],
+    })
+  ),
```

Comment:
nice, didn't even have to change the override format!

Reply:
yeah their doc didn't show it clearly but the typescript definitions were clear you could do this